src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
@jim-p In this case the rewriting provides no benefit.  And other files just as critical are not handled that way to ensure content either.  e.g. the CA bundle.  And recreating form config can just as easily create incorrect content.  There is no more protection of the config contents to be correct than the existing file.